### PR TITLE
Measured characteristic change

### PR DESCRIPTION
**Pull request by libertymayc**
Change in measured characteristic: alignment with Figma

### CODE REVIEW

**Review by joshwooding — CHANGES_REQUESTED** [review]
Missing a changeset

File: .changeset/plenty-cars-boil.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Change to measured characteristic: borderSize 1px -> 2px; borderStyle states added (active, complete, incomplete); foreground and borderColor tokens; disabled background token
```
Comment (joshwooding) [inline .changeset/plenty-cars-boil.md:6]: Could we expand on the changes to measured characteristic. Potentially in a similar format to: https://github.com/jpmorganchase/uitk/pull/589/files#diff-e88587ddac519bdb65910b0a821c5da1396994585d5f00905e46f1441054695f

**Comment by joshwooding** [comment]
@ZarkoPernar As liberty is out can you add the finishing touches here please

File: .changeset/plenty-cars-boil.md (hunk 1 above)
Comment (ZarkoPernar) [reply .changeset/plenty-cars-boil.md]: @joshwooding 
https://github.com/jpmorganchase/uitk/pull/532/commits/ef396742436f31a9f248bc9489712379dcbddd59

Comment (joshwooding) [reply .changeset/plenty-cars-boil.md]: Perfect :)

**Review by joshwooding — DISMISSED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
Same issue as https://github.com/jpmorganchase/uitk/pull/560

**Pull request merged**